dashboard/app/models/pd/workshop.rb

```diff
@@ -311,6 +311,15 @@ def year
     sessions.order(:start).first.start.strftime('%Y')
   end
 
+  # returns the school year the summer workshop is preparing for, in
+  # the form "2019-2020", like application_year on Pd Applications
+  def summer_workshop_school_year
+    if local_summer?
+      y = year
+      "#{y}-#{y.to_i + 1}"
```

Comment:
Given how many year ranges we have like this, seems like this might be a candidate for a more global helper function at some point in the future?